core/bootstrap.php

```diff
@@ -7,9 +7,13 @@
 require 'core/Request.php';
 require 'core/database/Connection.php';
 require 'core/database/PublisherQueryBuilder.php';
+require 'core/database/GamesQueryBuilder.php';
 
 $app['database'] = new PublisherQueryBuilder(
     Connection::make($app['config']['database'])
 );
+$appGames['database'] = new GamesQueryBuilder(
```

Comment:
pasivadink kaip nors normaliai tuos query buildierius, o ne $appGames['database'] :D ir po to parefactorink juos visur kitur - ismok naudoti phph stormo code refactor tools'a :)